src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import md5 from 'md5';
+import { trim } from 'lodash';
+/**
+ * Displays user profile avatar image with/withour user name
+ */
+const Avatar = ({ className, size, userName, imgUrl, userEmail }) => {
+  const gravatarUrl = userEmail
+    ? `https://www.gravatar.com/avatar/${md5(trim(userEmail.toLowerCase()))}?s=${size}`
+    : imgUrl;
+  const avatarClass = classes('Avatar', className);
+  const avatarSize = size >= 20 && size <= 200 ? size : 40;
+  const avatarLetterSize =
```

Comment:
It would be nice if we could also use utility classes to define the size, or perhaps through `className`, though I'm not sure I see a way to do that right now.

Reply:
I've also been thinking about updating the width and height utility classes to match the spacing system but thought that might be too restrictive